params-2025x.json

```diff
@@ -329,7 +420,15 @@
                     "max": 1,
                     "default": 0,
                     "uiOrder": 37,
-                    "group": "Volume"
+                    "group": "Volume",
+                    "requires": [
+                        "albedoColor != (0,0,0)",
+                        "transparency > 0",
+                        "thinWalled == false",
+                        "attenuationColor != (0,0,0)",
+                        "attenuationColor != (1,1,1)"
+                        "attenuationDistance > 0"
```

Comment:
In any case, the format should be documented in Chapter 6.4, Parameter JSON file format.